src/main/java/io/github/dgroup/enumerable4j/Enumerable.java

```diff
@@ -62,6 +62,20 @@ default boolean all(Predicate<X> prd) {
         return match;
     }
 
+    /**
+     * Passes each element of the collection to the each given blocks.
+     * @param first The predicate to match each element.
+     * @param other The array of predicates to match each element.
+     * @return The true if the blocks never return false or nil.
+     */
+    default boolean all(Predicate<X> first, Predicate<X>... other) {
```

Comment:
no need to add new methods, you should change old methods.
Please add everywhere `Predicate<X> ... other`

Reply:
done.